src/VirtualList/VirtualList.js

```diff
@@ -210,6 +215,16 @@ VirtualList.getItems = function (viewTop, viewHeight, listTop, itemHeight,
   return result;
 };
 
+VirtualList.getItemsToRender = function (props, state) {
```

Comment:
To make it testable, and following how this was built out in the first place. I wanted to not spend more time on this fix then necessary. If I make that change, I feel we should do it for the whole file.